src/main/java/org/dependencytrack/parser/snyk/SnykParser.java

```diff
@@ -56,14 +58,18 @@ public Vulnerability parse(JSONArray data, QueryManager qm, String purl, int cou
 
                 for (int countCoordinates = 0; countCoordinates < coordinates.length(); countCoordinates++) {
                     JSONArray representation = coordinates.getJSONObject(countCoordinates).optJSONArray("representation");
-                    if (representation != null) {
+                    if ((representation.length() == 1 && representation.get(0).equals("*"))) {
+                        LOGGER.warn("Range only contains *. Will not compute vulnerable software for this range. Purl is: "+purl);
```

Comment:
Please lower this to `debug`, as there is nothing users can do against it, and it produces a lot of noise.

Reply:
done